<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CV of Abdelmounim Essabbar">
    <title>Abdel Mounim Essabbar - Web Profile</title>
    <link rel="stylesheet" href="styles.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" />
    <!-- Bootstrap CSS -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
</head>
<body>
    <div class="container-a">
        <!-- Left panel (fixed) -->
        <div class="sidebar-a">
            <img src="your-image.jpg" alt="Profile Picture" class="profile-pic">
            <h1>Abdelmounim Essabbar</h1>
            <p class="role">Bioinformatician | Data Scientist</p>
            <div class="contact-info">
                <p><i class="fas fa-home"></i> Toulouse, France</p>
                <p><i class="fas fa-phone"></i> +33582741693 </p>
                <p><i class="fas fa-envelope"></i> <a href="mailto:mounim1024(a)gmail.com">mounim1024(a)gmail.com</a></p>
                <p><i class="fab fa-twitter"></i> <a href="https://twitter.com/mouneem">@mouneem</a></p>
                <p><i class="fab fa-github"></i> <a href="https://github.com/mouneem">GitHub</a></p>
                <p><i class="fab fa-linkedin"></i> <a href="https://linkedin.com/in/mouneem">LinkedIn</a></p>
                <p><i class="fas fa-flask"></i> <a href="https://scholar.google.com/citations?user=Io_FGRgAAAAJ">Google Scholar</a></p>
            </div>
        </div>

        <!-- Right panel (scrollable content) -->
        <div class="content">
            <!-- Biography Section -->
            <section class="biography">
                <h2>Biography</h2>
                <p>I am a bioinformatician with expertise in oncology, virology and genomic research. I work on multi-omics data integration and computational biology, with a focus on discovering novel biomarkers and therapeutic targets for cancer research. <br>
                   Currently, I am a Post-doc in Bioinformatics at CRCT (INSERM), Toulouse, focusing on Tumor MicroEnvironment (TME) profiling via single cell, spatial and immune-deconvolution analysis.</p>
            </section>

            <!-- Research Interests Section -->
            <section class="research-interests">
                <h2>Research Interests</h2>
                  <div class="card-container">
              
                      <!-- Card 1: Cancer Genomics -->
                      <div class="card card-1">
                          <div class="card-icon">
                            <i class="fa-solid fa-disease"></i>
                          </div>
                          <div class="card-content">
                              <h3>Computational Approaches for Cancer Genomics</h3>
                              <p>Exploring computational methods to advance cancer genomics and immuno-oncology research.</p>
                          </div>
                      </div>
              
                      <!-- Card 2: Single-cell RNA sequencing -->
                      <div class="card card-2">
                          <div class="card-icon">
                            <i class="fa-solid fa-dna"></i>
                          </div>
                          <div class="card-content">
                              <h3>Single-cell RNA Sequencing & Spatial Transcriptomics</h3>
                              <p>Utilizing single-cell RNA sequencing and spatial transcriptomics for deeper biological insights.</p>
                          </div>
                      </div>
              
                      <!-- Card 3: AI and Machine Learning -->
                      <div class="card card-3">
                          <div class="card-icon">
                            <i class="fa-solid fa-code"></i>
                          </div>
                          <div class="card-content">
                              <h3>AI & Machine Learning in Biomarker Discovery</h3>
                              <p>Applying AI and machine learning techniques to uncover new biomarkers for disease research.</p>
                          </div>
                      </div>
              
                      <!-- Card 4: Bioinformatics Pipelines -->
                      <div class="card card-4">
                          <div class="card-icon">
                            <i class="fa-solid fa-code"></i>
                          </div>
                          <div class="card-content">
                              <h3>Bioinformatics Pipelines for Deconvolution & GRNs</h3>
                              <p>Developing bioinformatics pipelines for deconvolution and gene regulatory networks (GRN).</p>
                          </div>
                      </div>
              
                      <!-- Card 5: High-Throughput Algorithms -->
                      <div class="card card-5">
                          <div class="card-icon">
                            <i class="fa-solid fa-database"></i>
                          </div>
                          <div class="card-content">
                              <h3>High-Throughput Data Analysis</h3>
                              <p>Creating algorithms to handle and analyze large-scale, high-throughput biological data.</p>
                          </div>
                      </div>
              
                  </div>
              
            </section>

            
            <!-- Professional Experiences Section -->
            <section class="experience">
                <h2>Professional Experiences</h2>
                <div class="job">
                  <div class="row">
                    <div class="col-2 text-center pt-3">
                      <img src="https://www.crct-inserm.fr/wp-content/uploads/2022/07/CRCT_logo_complet.png" height="75px" alt=""> <br>
                      <img src="https://fundit.fr/sites/default/files/styles/max_650x650/public/actors/2751-institut-national-sante-et-recherche-medicale-inserm.jpg?itok=3RH2o1ZW" height="75px" alt="">
                    </div>
                    <div class="col-9">
                      <h2>PostDoc / Bioinformatics Engineer</h2>
                      <p><strong>March 2022 - Present</strong>, <em>CRCT / INSERM, Toulouse, France</em></p>

                      <p>As part of my role, I lead various projects in the field of oncology, focusing on the integration and analysis of high-dimensional spatial transcriptomics data, immune profiling, and the development of new bioinformatics pipelines. My key contributions include:</p>

                      <ul>
                          <li>
                              <strong>Multi-Omics Analysis:</strong> I lead multi-omics data analysis efforts, specifically in the context of immune profiling and spatial transcriptomics, aiming to better understand tumor microenvironments and improve treatment outcomes for cancer patients. 
                          </li>
                          <li>
                              <strong>Development of Bioinformatics Pipelines:</strong> I have developed several bioinformatics pipelines for processing high-throughput data, including tools to integrate transcriptomics, imaging, and genetic mutations to reveal biomarkers associated with immunotherapy response in lung cancer.
                          </li>
                          <li>
                              <strong>Benchmarking Spatial Omics Tools:</strong> I benchmarked various tools, including MOSNA, UTAG, and CellCharter, to improve workflows for spatial omics data analysis. My work includes reconstructing cellular networks from imaging and spatial-omics data to extract biological insights related to therapy resistance.
                          </li>
                      </ul>

                      <h3>Key Projects</h3>
                      <ul>
                          <li>
                              <strong>IMMUcan (WP7: Data Integration and Bioinformatics):</strong> Integrated and analyzed complex datasets, applying both supervised and unsupervised approaches to extract key biomarkers predictive of cancer immunotherapy response.  
                          </li>
                          <li>
                              <strong>Benchmarking Network Approaches for Spatial Multi-Omics:</strong> Compared several methods for analyzing spatial data in the context of immuno-oncology. My work highlighted the importance of choosing the right method to extract meaningful insights from spatial-omics data, such as identifying immune niches in tumor microenvironments.
                          </li>
                      </ul>
                      
                      </ul>
                    </div>
                  </div>
                </div>
                <hr>
                <div class="job">

                  <div class="row">
                    <div class="col-2 text-center pt-3">
                      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSAhmyD_WGIS8B8SgGMKbEBRgc6o5I6YN_BxA&s" height="75px" alt=""> 
                      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQruUGZKqqzmcIKFZzDA0k4B4u9MMws0gdBYA&s" height="75px" alt="">
                    </div>
                    <div class="col-9">
                      <h3>Doctoral Candidate in Bioinformatics</h3>
                      <p><strong>2019 - 2022, Mohammed V University, Rabat, Morocco</strong></p>
                      <div class="phd-research">
                        <h2>Ph.D. Research Contributions</h2>
                        <p>During my Ph.D. research, I made significant contributions to understanding the evolution of <strong>SARS-CoV-2</strong> and its genetic variations, particularly during the global COVID-19 pandemic. My work emphasized the development of computational tools and databases to monitor and analyze large-scale viral genomic data. Some key achievements include:</p>
                    
                        <ul>
                            <li>
                                <strong>Development of Bioinformatics Tools:</strong> I developed <em>Bingo-3C</em>, an innovative in-house software for DNA sequence compression, comparison, and clustering. This tool efficiently minimized the time and memory needs for handling vast quantities of DNA sequence data by introducing a novel representation method. It proved essential in tracking the genetic changes in <strong>SARS-CoV-2</strong> sequences.
                            </li>
                    
                            <li>
                                <strong>Creation of IDbSV Database:</strong> I designed and managed the <em>IDbSV database</em>, a comprehensive repository of over <strong>60,000 genetic alterations</strong> from more than a million <strong>SARS-CoV-2 genomes</strong>. This database, publicly available since November 2020, enabled researchers globally to understand the sources and impacts of mutations in real-time, playing a crucial role in pandemic response efforts.
                            </li>
                    
                            <li>
                                <strong>Analysis of SARS-CoV-2 Evolution:</strong> My thesis presented a detailed analysis of the evolution of <strong>SARS-CoV-2</strong> over four years (2020-2023), summarizing key findings on <em>mutation hotspots</em>, <em>geographical spread</em>, and <em>temporal trends</em>. I monitored and characterized the virus’s mutation rate and evolutionary patterns, contributing to global efforts in tracking its spread and assisting in the design of public health strategies.
                            </li>
                    
                            <li>
                                <strong>Impact of the Work:</strong> The bioinformatics tools and databases developed during this period were instrumental in processing millions of genomes and summarizing crucial findings that aided in the international scientific community's understanding of the virus.
                            </li>
                        </ul>
                      </div>
                    
                    </div>
                  </div>
                </div>

                <div class="job">

                  <div class="row">
                    <div class="col-2 text-center pt-3">
                      <img src="https://flxflow.pt/wp-content/uploads/2021/02/Logo_igc.png" height="75px" alt=""> 
                    </div>
                    <div class="col-9">
                      <h2>Bioinformatics Engineer</h2>
                      <p><strong>2018 - 2019</strong>, <em>Systems Biology Laboratory, Instituto Gulbenkian de Ciência (IGC), Portugal</em></p>
                  
                      <p>As a Bioinformatics Engineer in the Systems Biology Laboratory, I contributed to multiple projects involving gene regulatory networks and computational simulations of biological systems. My key responsibilities and achievements include:</p>
                  
                      <ul>
                          <li>
                              <strong>Development of Computational Methods:</strong> I developed novel methods for analyzing <b>Gene Regulatory Network (GRN) models</b>, providing new insights into <b>cell signaling pathways</b>. These methods allowed for a more nuanced understanding of gene interactions and regulatory mechanisms in various biological processes.
                          </li>
                          <li>
                              <strong>Streamlining of Simulations:</strong> I automated the 
                              <b>simulation of signaling cascades </b> This automation improved the efficiency of research experiments and allowed for faster iterations in ongoing projects.
                          </li>
                          <li>
                              <strong>Mutation Impact Simulations:</strong> Conducted simulations to evaluate the impact of genetic mutations on biological functions, directly informing 
                              <b>experimental designs </b>. These simulations helped researchers predict the effects of mutations on cellular processes and guide laboratory experiments more effectively.
                          </li>
                      </ul>
                    </div>
                  </div>
                </div>

                
            </section>

            <!-- Key Research Projects Section -->
            <section class="projects">
                <h2>Key Research Projects</h2>
                <div class="project">
                  <h3>IMMUcan: Integrated iMMUnoprofiling of Large Cancer Patient Cohorts</h3>
                  <p>
                    Developed tools to profile immune cell interactions and discover biomarkers predictive of cancer immunotherapy responses. <br>
                    Read more: <a href="https://immucan.eu/about-the-project/" target="_blank"> https://immucan.eu/about-the-project/  <i class="fa fa-link" aria-hidden="true"></i> </a>
                  </p>
                </div>

                <div class="project">
                  <h3>LungPredict: Personalized Therapeutic Strategies for Lung Cancer</h3>
                  <p>
                    Developed bioinformatics tools to integrate molecular profiles of lung cancer patients and guide personalized treatments. <br>
                    Read more: <a href="https://www.crct-inserm.fr/en/lungpredict-en/" target="_blank"> https://www.crct-inserm.fr/en/lungpredict-en/  <i class="fa fa-link" aria-hidden="true"></i> </a>
                  </p>
                </div>
            </section>

            <!-- Publications Section -->
            <section class="publications">
                <h2>Publications</h2>


  <!-- Publication 5 -->
  <div class="publication">
    <b><a href="#" onclick="openModal(event, 'img5')">IDbSV: An Open-Access Repository for Monitoring SARS-CoV-2 Variations and Evolution</a></b>
    <p><strong>Cited by:</strong> 7 | <strong>Year:</strong> 2021 | <em>Frontiers in Medicine</em></p>
    <img id="img5" src="https://i1.rgstatic.net/publication/356997189_IDbSV_An_Open-Access_Repository_for_Monitoring_SARS-CoV-2_Variations_and_Evolution/links/61b86772fd2cbd72009befa1/largepreview.png" alt="Publication 5 Image" style="display: none;">
</div>



    <!-- Publication 1 -->
    <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img1')">Large scale genomic analysis of 3067 SARS-CoV-2 genomes reveals a clonal geo-distribution and a rich genetic variations of hotspots mutations</a></b>
      <p><strong>Cited by:</strong> 105 | <strong>Year:</strong> 2020 | <em>PloS One 15 (11), e0240345</em></p>
      <img id="img1" src="https://www.biorxiv.org/content/biorxiv/early/2020/05/03/2020.05.03.074567/F2.large.jpg" alt="Publication 1 Image" style="display: none;">
  </div>

  <!-- Publication 2 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img2')">Genomic diversity and hotspot mutations in 30,983 SARS-CoV-2 genomes: moving toward a universal vaccine for the “confined virus”?</a></b>
      <p><strong>Cited by:</strong> 75 | <strong>Year:</strong> 2020 | <em>Pathogens 9 (10), 829</em></p>
      <img id="img2" src="path_to_image2.jpg" alt="Publication 2 Image" style="display: none;">
  </div>

  <!-- Publication 3 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img3')">Genome sequences of six SARS-CoV-2 strains isolated in Morocco, obtained using Oxford Nanopore MinION technology</a></b>
      <p><strong>Cited by:</strong> 20 | <strong>Year:</strong> 2020 | <em>Microbiology Resource Announcements 9 (32), 10.1128/mra.00767-20</em></p>
      <img id="img3" src="path_to_image3.jpg" alt="Publication 3 Image" style="display: none;">
  </div>

  <!-- Publication 4 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img4')">Genetic diversity and evolutionary dynamics of the Omicron variant of SARS-CoV-2 in Morocco</a></b>
      <p><strong>Cited by:</strong> 7 | <strong>Year:</strong> 2024 | <em>Pathogens and Global Health 118 (3), 241-252</em></p>
      <img id="img4" src="path_to_image4.jpg" alt="Publication 4 Image" style="display: none;">
  </div>

  <!-- Publication 6 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img6')">Report of SARS-CoV-2 B1.1.7 lineage in Morocco</a></b>
      <p><strong>Cited by:</strong> 7 | <strong>Year:</strong> 2021 | <em>Microbiology Resource Announcements 10 (16), 10.1128/mra.00240-21</em></p>
      <img id="img6" src="path_to_image6.jpg" alt="Publication 6 Image" style="display: none;">
  </div>

  <!-- Publication 7 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img7')">Genetic analysis of SARS-CoV-2 strains collected from North Africa: viral origins and mutational spectrum</a></b>
      <p><strong>Cited by:</strong> 6 | <strong>Year:</strong> 2020 | <em>bioRxiv, 2020.06.30.181123</em></p>
      <img id="img7" src="path_to_image7.jpg" alt="Publication 7 Image" style="display: none;">
  </div>

  <!-- Publication 8 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img8')">In silico logical modelling to uncover cooperative interactions in cancer</a></b>
      <p><strong>Cited by:</strong> 5 | <strong>Year:</strong> 2021 | <em>International Journal of Molecular Sciences 22 (9), 4897</em></p>
      <img id="img8" src="path_to_image8.jpg" alt="Publication 8 Image" style="display: none;">
  </div>

  <!-- Publication 9 -->
  <div class="publication">
      <b><a href="#" onclick="openModal(event, 'img9')">Draft genome sequence of Stenotrophomonas maltophilia MDMC339, isolated from soil of Merzouga desert in Morocco</a></b>
      <p><strong>Cited by:</strong> 4 | <strong>Year:</strong> 2020 | <em>Microbiology Resource Announcements 9 (32), 10.1128/mra.00634-20</em></p>
      <img id="img9" src="path_to_image9.jpg" alt="Publication 9 Image" style="display: none;">
  </div>


              
              <!-- Modal structure -->
              <div id="modal" class="modal">
                  <span class="close" onclick="closeModal()">&times;</span>
                  <img class="modal-content" id="modal-image">
                  <div id="caption"></div>
              </div>


            </section>

            <!-- Skills Section -->
            <section class="skills">
                <h2>Skills</h2>
                <ul>
                    <li><strong>Bioinformatics:</strong> Spatial omics, single-cell RNA sequencing, genomics, transcriptomics, immune-deconvolution</li>
                    <li><strong>Programming Languages:</strong> Python, R, SQL, PHP</li>
                    <li><strong>Web Frameworks:</strong> Django, Symfony, Laravel</li>
                    <li><strong>Data Analysis:</strong> Pandas, Tensorflow, Keras, Seaborn, Sklearn</li>
                    <li><strong>Development Environment:</strong> Linux, Git, Conda, Docker</li>
                </ul>
            </section>
        </div>
    </div>

    <script src="https://kit.fontawesome.com/a076d05399.js"></script>

<!-- Add JavaScript -->
<script>
  function openModal(event, imageId) {
      event.preventDefault();  // Prevent the default link behavior
      var modal = document.getElementById("modal");
      var modalImg = document.getElementById("modal-image");
      
      // Get the corresponding hidden image element and assign it to the modal
      var img = document.getElementById(imageId);
      modalImg.src = img.src;
      modal.style.display = "block";
  }

  function closeModal() {
      document.getElementById("modal").style.display = "none";
  }
</script>



</body>
</html>
